This adds definitions for I2C1 and I2C2 to the build for the NUCLEO-F…

hybotix opened the pull request:
This adds definitions for I2C1 and I2C2 to the NUCLEO_F429ZI board.

Commit:
This adds definitions for I2C1 and I2C2 to the build for the NUCLEO-F429ZI board.

hybotics commented:
I did not add any code.

I just edited the mpconfigboard.h file to add definitions for I2C1 and I2C2 to the NUCLEO-F429ZI build.
I did find some trailing whitespace that I just deleted.

Commit:
Signed-off-by: Dale Weber <hybotics.sd@gmail.com>

This adds definitions for I2C1 and I2C2 to the NUCLEO_F429ZI board.
PATH: ports/stm32/boards/NUCLEO_F429ZI/mpconfigboard.h

Commit:
Signed-off-by: Dale Weber <hybotics.sd@gmail.com>

This adds definitions for I2C1 and I2C2 for the NUCLEO_F429ZI.
PATH: ports/stm32/boards/NUCLEO_F429ZI/mpconfigboard.h

Commit:
Signed-off-by: Dale Weber <hybotics.sd@gmail.com>

Adds I2C1 and I2C2 to the NUCLEO-F429ZI, and better alignment.
PATH: ports/stm32/boards/NUCLEO_F429ZI/mpconfigboard.h

hybotics commented:
I installed pre-commit but now am getting:

$ git commit -s
[INFO] Installing environment for local.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/python3', '-mvirtualenv', '/home/hybotics/.cache/pre-commit/repo3c08ox3u/py_env-python3')
return code: 1
expected return code: 0
stdout:
    ModuleNotFoundError: No module named 'virtualenv.activation.xonsh'
    
stderr: (none)
Check the log at /home/hybotics/.cache/pre-commit/pre-commit.log

robert-hh commented:
The problem was with the commit message format. It has to be:

topic: This is a short description.

with the colon after topic, a capital letter at the start of the description, a full stop at the end and not longer than 75 characters,
